packages/admin/cms-admin/src/dam/DataGrid/selection/DamMoreActions.tsx

```diff
@@ -0,0 +1,108 @@
+            >
+                <MenuList>
+                    <MenuItem
+                        onClick={() => {
+                            damSelectionActionsApi.downloadSelected();
+                            handleClose();
+                        }}
+                    >
+                        <ListItemIcon>
+                            <Download />
+                        </ListItemIcon>
+                        <ListItemText primary={<FormattedMessage id="comet.dam.moreActions.downloadSelected" defaultMessage="Download selected" />} />
+                    </MenuItem>
+                    <StyledDivider />
+                    <MenuItem
+                        onClick={() => {
+                            damSelectionActionsApi.moveSelected();
+                            handleClose();
+                        }}
+                    >
+                        <ListItemIcon>
+                            <Move />
+                        </ListItemIcon>
+                        <ListItemText primary={<FormattedMessage id="comet.dam.moreActions.moveItems" defaultMessage="Move items" />} />
+                    </MenuItem>
+                    <MenuItem
+                        onClick={() => {
+                            damSelectionActionsApi.archiveSelected();
+                            handleClose();
+                        }}
+                    >
+                        <ListItemIcon>
+                            <Archive />
+                        </ListItemIcon>
+                        <ListItemText primary={<FormattedMessage id="comet.dam.moreActions.archiveItems" defaultMessage="Archive items" />} />
+                    </MenuItem>
+                    <MenuItem
+                        onClick={() => {
+                            damSelectionActionsApi.restoreSelected();
+                            handleClose();
+                        }}
+                    >
+                        <ListItemIcon>
+                            <Restore />
+                        </ListItemIcon>
+                        <ListItemText primary={<FormattedMessage id="comet.dam.moreActions.restoreItems" defaultMessage="Restore items" />} />
+                    </MenuItem>
+                    <MenuItem
+                        onClick={() => {
+                            damSelectionActionsApi.deleteSelected();
+                            handleClose();
+                        }}
+                    >
+                        <ListItemIcon>
+                            <Delete />
+                        </ListItemIcon>
+                        <ListItemText primary={<FormattedMessage id="comet.dam.moreActions.deleteItems" defaultMessage="Delete items" />} />
+                    </MenuItem>
+                </MenuList>
+            </Menu>
+        </>
+    );
+};
+
+const StyledDivider = styled(Divider)`
+    &.MuiDivider-root {
+        border-color: ${({ theme }) => theme.palette.grey["50"]};
```

Comment:
Is it really necessary to override the default divider color? Could we check this with UX?

Reply:
I used the color from the screen design, so I suppose UX is okay with that